### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by Muennighoff**
Fixes https://github.com/allenai/OLMo/issues/457

As this is a breaking change, it probably makes sense to release a new version with it

### CODE REVIEW

**Comment by dirkgr** [comment]
@epwalsh , @AkshitaB , @2015aroras , how much stuff will break if we merge this? Is it risky?

**Comment by dirkgr** [comment]
I agree with you on the version, so I bumped the version number in this PR.

**Comment by epwalsh** [comment]
I don't think trainer code or checkpoints will break, so I'm okay with it. Better to make this change now anyway before the next big run.

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added the option to directly pass input embeddings to `OLMo` and `OLMoForCausalLM`.
 - Added support for Python 3.8.
 - Added code to throw an error if `output_attentions` is set to `True` in forward call to `OLMoForCausalLM`. This functionality hasn't been implemented yet.
+- Rename `Olmo` to `OLMo` everywhere in the codebase
```
Comment (2015aroras) [inline CHANGELOG.md:18]: Please move this to Unreleased or v0.3.0 or similar

**Review by 2015aroras — COMMENTED** [review]
The only thing I can foresee breaking is trying to run OLMo via HF. I think that's probably inevitable for a small period of time.

**Review by dirkgr — APPROVED** [review]

**Pull request merged**